src/javascript/node-project.ts

```diff
@@ -833,6 +867,16 @@ export class NodeProject extends GitHubProject {
     this.npmignore?.addPatterns(pattern);
   }
 
+  /**
+  * Defines Prettier ignore Patterns
+  * these patterns will be added to the file .prettierignore
+  *
+  * @param {string} pattern filepatterns so exclude from prettier formatting
```

Comment:
type is already annotated in the code:

```suggestion
  * @param pattern filepatterns so exclude from prettier formatting
```